lib/oban/migrations.ex

```diff
@@ -138,7 +138,8 @@ defmodule Oban.Migrations do
     import Oban.Migrations.Helper
 
     def up(prefix) do
-      execute "CREATE SCHEMA IF NOT EXISTS #{prefix}"
+      if prefix != "public", do:
```

Comment:
This upsets the formatter. Moving it onto a single line should do it.

Reply:
Sorry about that, just updated.